### DISCUSSION TITLE
Please help us Upvote Astroid Framework on Product Hunt

### DISCUSSION THREAD

**Discussion by sonvnn** [templaza/astroid-framework#517] [Announcements]
Please help us share your experiences about the Astroid Framework.
https://www.producthunt.com/posts/astroid-framework

Thank you very much!
Cheers,
Sonny

**Comment by salahjaafar** [comment]
Already completed

https://www.producthunt.com/posts/astroid-framework?comment=3063941

**Reply by sonvnn** [reply]
Thank you :)

**Comment by vtheod** [comment]
Done :)
https://www.producthunt.com/posts/astroid-framework?comment=3065907

**Comment by southpaw3229** [comment]
done... thanks for all your hard work.  https://www.producthunt.com/posts/astroid-framework?comment=3081146

**Comment by Chacapamac** [comment]
Well, just remark I send my comment tor the Joomdev ProductHunt Profile. GRRRRRR!
 I remove it and post a better post the the **real ASTROID**. 🚀
https://www.producthunt.com/posts/astroid-framework?comment=3084273

**Comment by Tomisslaw** [comment]
Done.
